### ISSUE TITLE
Incorrect type definition for StyleDictionary.transformGroup

### ISSUE THREAD

**Issue by isaiahtaylor** [amzn/style-dictionary#697]
Version: style-dictionary 3.0.2

This sample:

```js
const StyleDictionary = require('style-dictionary');

console.log(StyleDictionary.transformGroup['scss']);
```

Outputs:

```
[
    'attribute/cti',
    'name/cti/kebab',
    'time/seconds',
    'content/icon',
    'size/rem',
    'color/css'
  ]
```

That would imply `transformGroup` has a type of `Record<string, string[]>`.

However, its current definition is:

https://github.com/amzn/style-dictionary/blob/65424772f5c5aa935fde53753c9bf6376c1e635e/types/index.d.ts#L64

And the definition of `TransformGroup`:
https://github.com/amzn/style-dictionary/blob/65424772f5c5aa935fde53753c9bf6376c1e635e/types/TransformGroup.d.ts#L14-L17

This causes an error in Typescript when you try to grab elements of that array.

**Comment by dbanksdesign** [comment]
Thanks for raising this @isaiahtaylor I just made a PR fixing this bug, #729 if you'd like to take a look.

**Comment by chazzmoney** [comment]
Fixed in #729
